src/components/atoms/picker-sheet/__tests__/index.test.web.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, fireEvent } from 'react-native-testing-library';
+
+// Explictly target .web until https://github.com/nearform/polaris/issues/42 is done
+import PickerSheet from '../index';
+
+const mocks = {
+  onValueChange: jest.fn(),
+  options: ['a', 'b', 'c', 'd'].map(item => ({ value: item })),
+  currentOption: { value: 'a' }
+};
+
+describe('Picker Sheet', () => {
+  it('value change calls onValueChange function', () => {
```

Comment:
If I use the `it()` function to define a test, I usually change the test's name/description to include `it` as the subject of the sentence, to express the intention of the test  (like in the e2e test: `it('changes the app language')`). 

Another option, would be leave the name/description as is, and use `test()` instead.

Reply:
Good call, consistency is good, I'll change it